docker-for-mac/docker-toolbox.md

```diff
@@ -160,7 +178,61 @@ is running a different version of server.
 
 You might also run into a similar situation with Docker Universal Control Plane (UCP).
 
-There are a few ways to address this problem and keep using your older machines. One solution is to use a version manager like [DVM](https://github.com/getcarina/dvm).
+There are a few ways to address this problem and keep using your older
+machines. One solution is to use a version manager like
+[DVM](https://github.com/getcarina/dvm).
+
+## Migrating from Docker Toolbox to Docker for Mac
+
+Docker for Mac does not proposeToolbox image migration as part of the Docker for
+Mac (D4M for short) installer since version 18.01.0.  Migrating existing Docker
```

Comment:
I don't think D4M is appropriate branding, please remove it.

Avoid double-spacing between sentences.

Reply:
Wow, it's going to be hard for this one.  Used to it for years...

Reply:
You can call it whatever you want in your head but Docker for Mac is the official name. ;)

Reply:
I was referring to the single space after the period.

Reply:
Aha! Yes, that is a [convention because of typewriters!](https://en.wikipedia.org/wiki/Sentence_spacing#Mechanical_type_and_the_advent_of_the_typewriter). 

I was making a joke about acronyms we get used to using. :)